src/rust-crypto/backup.ts

```diff
@@ -156,6 +156,9 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
             );
 
             await this.olmMachine.saveBackupDecryptionKey(backupDecryptionKey, backupCheck.backupInfo.version);
+            // Emit an event that we have a new backup decryption key, so that the sdk can start
+            // importing keys from backup if needed.
+            this.emit(CryptoEvent.KeyBackupDecryptionKeyCached, backupCheck.backupInfo.version);
```

Comment:
maybe we should add a `saveDecryptionKey` method to `RustBackupManager` which calls `OlmMachine.saveBackupDecryptionKey` and emits the event, for a bit less copy-pasting?

Reply:
Yes, thought about and got lazy. Now done